### ISSUE TITLE
Slack bot image url returns 404

### ISSUE THREAD

**Issue by strifel** [zammad/zammad#3686]
### Infos:

* Used Zammad version: -
* Installation method (source, package, ..): -
* Operating system: -
* Database + version: -
* Elasticsearch version: -
* Browser + version: -

Issue found in Code in 3b8ff4d17848a0a12c3486463316b5d661301b84

### Expected behavior:

* The URL should provide a logo for the Slack Webhook bot

### Actual behavior:

* The URL for the image returns 404

### Steps to reproduce the behavior:

* Open: https://zammad.com/assets/images/logo-200x200.png

Yes I'm sure this is a bug and no feature request or a general question.

### Code: 

https://github.com/zammad/zammad/blob/916bb77066316ede9afdbc16ab9d177f83ea1f14/app/models/transaction/slack.rb#L164

**Comment by MrGeneration** [comment]
Please use the complete issue template.

**Comment by strifel** [comment]
@MrGeneration Thanks for the hint.
I used Githubs "Create an issue referencing this line" and it did somehow not use the issue template.

I updated the Issue to use the template

**Comment by MrGeneration** [comment]
@martini  not sure if this file every existed von `zammad.com` - if it did it would be the easiest to put it back in place.

**Comment by davidc** [comment]
Here is a 200x200 PNG logo
![logo-200x200](https://user-images.githubusercontent.com/1050152/131996079-cc095c2b-19ed-4677-9513-a0b1ee8035c0.png)

**Comment by davidc** [comment]
![logo-200x200](https://user-images.githubusercontent.com/1050152/131996256-32521e9c-a9cb-47db-80df-883305a0e333.png)

Here is a 200x200 PNG logo

**Comment by cbrand-ludlums** [comment]
To add to this, if you specify a custom icon_url from the slack integration setup page, it is ignored and uses the currently missing  (https://zammad.com/assets/images/logo-200x200.png) instead.  I modified line 164 of file:

/opt/zammad/app/models/transaction/slack.rb

and changed it from
`logo_url = 'https://zammad.com/assets/images/logo-200x200.png'`

to
`logo_url = 'https://your_custom_icon_url .png'`

restarted the zammad service and it fixed it so it now uses my custom icon_url.  From the code at that location, it looks like the variable local_config['logo_url'] isn't being set somewhere from the setting in the slack integration config, so it's always defaulting to the hardcoded icon_url from the zammad.com website.

I'm using Mattermost instead of slack.  I had noticed a similar issue and posted about it in the community forum a while back and tingeber pointed me to the right file to change.
https://community.zammad.org/t/mattermost-integration-icon-url/6831

My fix is only temporary and will have to be redone after each update, but it's working like I was wanting.

best regards!
